### PR TITLE
add tests GRA.9 and GRA.10

### PR DESCRIPTION
**Pull request by redtechnologies**

### CODE REVIEW

File: src/harness/testcases/grant_testcase.py [hunk 1]
```diff
@@ -161,4 +161,70 @@ def test_WINFF_FT_S_GRA_7(self):
...
+
+    # Request grant
+    grant_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_0['cbsdId'] = cbsd_id
+    del grant_0['operationParam']['maxEirp']
+    request = {'grantRequest': [grant_0]}
+    response = self._sas.Grant(request)['grantResponse'][0]
+    # Check grant response
+    self.assertFalse('cbsdId' in response)
```
Comment (qingwu123) [inline src/harness/testcases/grant_testcase.py]: Test plan says "verify cbsdId (Response) = C"?

File: src/harness/testcases/grant_testcase.py [hunk 2]
```diff
@@ -161,4 +161,70 @@ def test_WINFF_FT_S_GRA_7(self):
...
+
+    # Request grant
+    grant_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_0['cbsdId'] = cbsd_id
+    del grant_0['operationParam']['operationFrequencyRange']['lowFrequency']
+    request = {'grantRequest': [grant_0]}
+    response = self._sas.Grant(request)['grantResponse'][0]
+    # Check grant response
+    self.assertFalse('cbsdId' in response)
```
Comment (qingwu123) [inline src/harness/testcases/grant_testcase.py]: Test plan says "verify cbsdId (Response) = C"?

**Comment by redtechnologies** [comment]
I just pushed the latest modification with the corrected code for both GRA.9 and GRA.10
I wasn't quite sure how to write the tests because the GRA.8 test which is currently on the master branch as  "test_10_7_4_1_3_1_1_2" is also supposed to check the grant response for the cbsdId according to the spec but instead does the following "self.assertFalse('cbsdId' in response)".

**Comment by kate-harrison** [comment]
I agree that test_10_7_4_1_3_1_1_2 is incorrectly written right now. Would you be willing to submit a PR to fix the problem and rename the test?

**Pull request merged**